src/machine-learning/machine-learning.ts

```diff
@@ -76,6 +79,44 @@ export class MachineLearning implements FirebaseServiceInterface {
       });
     }
 
+    let storage: typeof StorageClient;
```

Comment:
Just use `app.storage()` here and remove the dependency check. Let's also avoid storing a reference to storage at the class level. Just call `appInternal.storage()` where you need it.

Reply:
done